stackslib/src/net/stackerdb/sync.rs

```diff
@@ -1030,7 +1043,8 @@ impl<NC: NeighborComms> StackerDBSync<NC> {
                 .1
                 .iter()
                 .enumerate()
-                .find(|(_i, naddr)| !self.comms.has_inflight(naddr));
+                // .find(|(_i, naddr)| !self.comms.has_inflight(naddr));
```

Comment:
Is this intended to be put back?

Reply:
I don't think so -- but this is a question for @jcnelson I think

Reply:
It seems like it should, or else there is no need to `find`.

Reply:
yeah drop it

Reply:
Removed in 53855fecdd7f91c28199a47858571dd32fc18259